lib/generator.js

```diff
@@ -22,4 +25,32 @@ module.exports = function(locals){
 
     return result.concat(data);
   }, []);
+
+  // generate tag index page, usually /tags/index.html
+  if(tagConfig.enable_index_page) {
+    tagDir = config.tag_dir;
+    if (tagDir[tagDir.length - 1] !== '/') {
+     tagDir += '/';
+    }
+    pages.push({
+      path: tagDir,
+      layout: [tagConfig.template ? tagConfig.template : 
+        'tagIndex', 'tag', 'archive', 'index'],
```

Comment:
You can use:

``` js
[].concat(tagConfig.template, ['tagIndex', 'tag', 'archive', 'index']);
```

In this way, users will still have the template fallback when the template does not exist. Also I think `tag-index` would be better than `tagIndex`.